src/poly/domain.rs

```diff
@@ -177,6 +177,14 @@ impl<G: Group> EvaluationDomain<G> {
         }
     }
 
+    /// Returns a constant polynomial in the Lagrange coefficient basis
+    pub fn const_lagrange(&self, scalar: G) -> Polynomial<G, LagrangeCoeff> {
```

Comment:
Rename this to `constant` because `const` is a keyword in Rust that indicates the function can be evaluated at compile time; that is not the case here.